src/cashweb/relay/index.ts

```diff
@@ -575,11 +575,8 @@ export class RelayClient extends ReadOnlyRelayClient {
       // Ensure all outpoints are on-chain before trying to send message. Don't
       // want to burn other transactions if our state is out of sync with the
       // blockchain.
-      Promise.all(
-        stagedUtxos.map(async outpoint => {
-          return wallet.checkOutpoint(outpoint)
-        }),
-      )
+      wallet
+        .checkAreOutpointsUnspent(stagedUtxos)
```

Comment:
Sometimes things get out of sync here. There needs to be a way to delete the broken UTXOs. This function doesn't seem to do it anymore.

Reply:
Basically, this thing can get stuck in a loop of being unable to send messages if something doesn't clean up the local state.